rizza/helpers/config.py

```diff
@@ -101,7 +138,14 @@ def load_config(self, cfg_file=None):
             self._load_genetics()
             self.RIZZA["LOG PATH"] = self.RIZZA.get("LOG PATH", "logs/rizza.log")
             if self.RIZZA["LOG PATH"] != str(Path(self.RIZZA["LOG PATH"]).absolute()):
-                self.RIZZA["LOG PATH"] = self.base_dir.joinpath(self.RIZZA["LOG PATH"])
+                self.RIZZA["LOG PATH"] = str(self.base_dir.joinpath(self.RIZZA["LOG PATH"]))
```

Comment:
Refactor all of these defaults to be a constant dictionary at the top of the file that will be used to populate the config values if they are missing.